service/history/shard/context_impl.go

```diff
@@ -51,6 +51,8 @@ import (
 
 var (
 	defaultTime = time.Unix(0, 0)
+
+	persistenceOperationRetryPolicy = common.CreatePersistanceRetryPolicy()
```

Comment:
could we possibly fix the spelling of this?  It should be CreatePersistenceRetryPolicy()

Reply:
Sure